.githooks/prepare-commit-msg

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# If the branch name contains the issue number, it will append it to the
+# commit message. Example:
+#
+#   BRANCH NAME               APPEND
+#   feature/PLANET-3125-fix   PLANET-3125
+#   planet-3125               PLANET-3125
+#   some-name                 fail
+
+
+# Customize which branches should be skipped when
+# prepending commit message.
+if [ -z "$BRANCHES_TO_SKIP" ]; then
+  BRANCHES_TO_SKIP=(master develop)
+fi
+
+
+BRANCH_NAME=$(git symbolic-ref --short HEAD)
+PLANET_BRANCH_NAME=$(git symbolic-ref --short HEAD | grep -Po "(PLANET|planet)-[0-9]*" |  tr '[:lower:]' '[:upper:]')
+BRANCH_NAME="${BRANCH_NAME##*/}"
+BRANCH_EXCLUDED=$(printf "%s\n" "${BRANCHES_TO_SKIP[@]}" | grep -c "^$BRANCH_NAME$")
+ISSUE_IN_COMMIT=$(grep -cP "(PLANET|planet)-[0-9]*" $1)
+
+# If planet-xxx is not included in the branch name or the commit message quit.
+# If it is included in the branch name and not in the commit message, prepend it to the commit message.
+if ! [[ -z "$PLANET_BRANCH_NAME" ]] && ! [[ $BRANCH_EXCLUDED -eq 1 ]] && ! [[ $ISSUE_IN_COMMIT -ge 1 ]]; then
+  sed -i.bak -e "1s/^/$PLANET_BRANCH_NAME /" $1;
```

Comment:
```
In .githooks/prepare-commit-msg line 28:
  sed -i.bak -e "1s/^/$PLANET_BRANCH_NAME /" $1;
                                             ^-- SC2086: Double quote to prevent globbing and word splitting.
```